packages/keychain/src/address-derivation/index.ts

```diff
@@ -19,8 +19,7 @@ export function deriveStxAddressChain(chain: ChainID) {
     if (!childKey.privateKey) {
       throw new Error('Unable to derive private key from `rootNode`, bip32 master keychain');
     }
-    const ecPair = ECPair.fromPrivateKey(childKey.privateKey);
```

Comment:
`ECPair.fromPrivateKey( ...` without options provided, would implicitly compress the passed private key